ads/_config.js

```diff
@@ -127,6 +127,14 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  admixer: {
+    renderStartImplemented: true,
```

Comment:
When I test with your ad examples, I couldn't get the render-start message. Could you confirm that renderStartAPI is implemented? Same with noContentAvailable API.